lc56.py

```diff
@@ -1 +1,24 @@
+class Solution:
+    def merge(self, intervals: List[List[int]]) -> List[List[int]]:
+        def isOverlap(a,b):
+            maxStart = max(a[0],b[0])
+            minFinish = min(a[1],b[1])
+            return (minFinish >=  maxStart) # overlap
+        
+        intervals.sort(key = lambda x:x[0])
+        res = []
+        temp = intervals[0]
+        res.append(temp)
+        
+        for i in range(1,len(intervals)):
```

Comment:
You can also do `for interval in intervals{1:]` (takes up some space though but on the same order as res so should be ok). No space version: `for interval in islice(intervals, 1, None)`